sdk/identity/Azure.Identity.Broker/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.2.0-beta.1 (2024-04-17)
 
 ### Other Changes
+- The `IsProofOfPossessionRequired` property on `InteractiveBrowserCredentialBrokerOptions` and `SharedTokenCacheCredentialBrokerOptions` has been removed. This property has moved to the experimental `PopTokenRequestContext` struct.
```

Comment:
Is this a breaking change?

Reply:
no, because it has never been GA'd

Reply:
it's an overload method. so may be not?

Reply:
I'll switch it to the breaking change section, just to be safe